LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2021 Northern.tech AS
+Copyright 2022 Northern.tech AS
 
```

Comment:
Unrelated - better if we can get this (and LIC_FILES_CHKSUM change) on a separate commit.